client/modules/IDE/pages/MobileIDEView.jsx

```diff
@@ -104,9 +58,23 @@ const IDEViewMobile = (props) => {
           <h2>{project.name}</h2>
           <h3>{selectedFile.name}</h3>
         </div>
+
+        <div style={{ marginLeft: '2rem' }}>
+          <Link to="/mobile/preferences">
+            <IconButton onClick={() => setOverlay('preferences')}>
+              <PreferencesIcon focusable="false" aria-hidden="true" />
```

Comment:
You should import this icon to  ` client/common/icons` so you don't have to manually put in the accessibility attributes! Same with the `PlayIcon`.